src/mtype.d

```diff
@@ -4542,103 +4542,23 @@ extern (C++) class TypeArray : TypeNext
         }
         if (ident == Id.reverse && (n.ty == Tchar || n.ty == Twchar))
```

Comment:
Are these the only places where `Id.reverse` and `Id.sort` are used?

Perhaps add a comment to set a future date for error -> gone.